cdec/update-server-packaging-war/src/main/java/com/codenvy/im/update/UtilService.java

```diff
@@ -41,15 +43,17 @@
     public UtilService() {
     }
 
-    /** Get client's IP. */
+    /**
+     * Get client's IP.
```

Comment:
It is used to identify anonymous user in "Weekly Codenvy on-prem" report and for statistics.